### PR TITLE
📝 Rework region migration instructions

### LINKED ISSUE #2418: 📝 Rework region migration
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/projects/region-migration.html

### What exactly should be updated?

Update for:

- Content accuracy
- Includes what it should
- Doesn't include what doesn't fit
- Fits template
- Fits style guide

Ensure reasons for migration are up to date. Including what a legacy region is.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2418

## What's changed

Based on #2434 
Redid the page to fit the style guide and template

### CODE REVIEW

File: docs/src/development/regions.md [hunk 1]
```diff
@@ -34,8 +34,21 @@ See all available information in the [API documentation](https://api.platform.sh
 
 ## Region availability
 
-The regions listed here may be different from those available when you create a new project.
-The list includes legacy regions as reference for existing projects.
+The regions listed here may be different from those available to you when you create a new project.
+Each organization can have its own rules for what regions to allow
```
Comment (trolologuy) [inline docs/src/development/regions.md:38]: :clipboard:
```suggestion
Each organization can have its own rules for what regions to allow.
```

File: docs/src/development/regions.md [hunk 2]
```diff
@@ -34,8 +34,21 @@ See all available information in the [API documentation](https://api.platform.sh
...
+When adding a new project, you only see regions allowed by your organization.
+
+The list of regions includes legacy regions as reference.
+
+### Legacy regions
+
+A legacy region is a region running an older version of the Platform.sh orchestration system.
+It doesn't include all available features.
+See a [list of not included features](../projects/region-migration.md#why-migrate-between-regions)
```
Comment (trolologuy) [inline docs/src/development/regions.md:47]: :clipboard:
```suggestion
See a [list of not included features](../projects/region-migration.md#why-migrate-between-regions).
```

**Review by trolologuy — CHANGES_REQUESTED** [review]
A couple typos, but other than that, great change! The migration guide is much better now 😍 

- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.

File: docs/src/development/regions.md (hunk 1 above)
Comment (CollierCZ) [inline docs/src/development/regions.md:38]:
```suggestion
Each organization can have its own rules for what regions to allow.
```

**Review by trolologuy — APPROVED** [review]
✅

**Pull request merged**